addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -0,0 +1,244 @@
+ 
+**Examples:**  
+Assumed you mapped the virtual datapoint to a String item called Display_Options   
+```
+String Display_Options "Display_Options" { channel="homematic:HM-RC-19-B:ccu:KEQ0099999:V_18_DISPLAY_OPTIONS" }
+```
+show message TEST:
+```
+smarthome send Display_Options "TEST"
+```
+show message TEXT, beep once and turn backlight on:
+```
+smarthome send Display_Options "TEXT, TONE1, BACKLIGHT_ON"
+```
+ 
+show message 15, beep once, turn backlight on and shows the celsius unit:
+```
+smarthome send Display_Options "15, TONE1, BACKLIGHT_ON, CELSIUS"
+```
+ 
+show message ALARM, beep three times, let the backlight blink fast and shows a bell symbol:
+```
+smarthome send Display_Options "ALARM, TONE3, BLINK_FAST, BELL"
+```
+ 
+Duplicate options: TONE3 is ignored, because TONE1 is specified previously.
+```
+smarthome send Display_Options "TEXT, TONE1, BLINK_FAST, TONE3"
+```
+
+### Troubleshooting
+**SHORT & LONG_PRESS events of push buttons do not occur on the event bus**  
+
+It seems buttons like the HM-PB-2-WM55 do just send these kind of events to the CCU if they are mentioned in a CCU program. A simple workaround to make them send these events is, to create a program (rule inside the CCU) that does just have a "When" part and no "Then" part, in this "When" part each channel needs to be mentioned at least once. As the HM-PB-2-WM55 for instance has two channels, it is enough to mention the SHORT_PRESS event of channel 1 & 2. The LONG_PRESS events will work automatically as they are part of the same channels. After the creation of this program, the button device will receive configuration data from the CCU which have to be accepted by pressing the config-button at the back of the device.
+
+**INSTALL_TEST**  
+
+If a button is still not working and you do not see any PRESS_LONG / SHORT in your log file (loglevel DEBUG), it could be because of enabled security. Try to disable security of your buttons in the HomeMatic Web GUI and try again. If you can't disable security try to use key INSTALL_TEST which gets updated to ON for each key press
+
+**-1 Failure**  
+
+A device may return this failure while fetching the datapoint values. I've tested pretty much but i did not found the reason. The HM-ES-TX-WM device for example always returns this failure, it's impossible with the current CCU2 firmware (2.15.2) to fetch the values. I've implemented two workarounds, if a device returns the failure, workaround one is executed, if the device still returns the failure, workaround two is executed. This always works in my tests, but you may see a OFFLINE, ONLINE cycle for the device.  
+Fetching values is only done at startup or if you trigger the RELOAD_FROM_GATEWAY virtual datapoint. I hope this will be fixed in one of the next CCU firmwares.  
+With [Homegear](https://www.homegear.eu) everything works as expected.
+
+### Debugging and Tracing
+
+If you want to see what's going on in the binding, switch the loglevel in logback.xml to DEBUG.
```

Comment:
Please note: Once https://github.com/openhab/openhab2/issues/194 is merged, we won't have a logback.xml anymore - you might want to remove this section for the moment.